dashboard/lib/projects_list.rb

```diff
@@ -0,0 +1,41 @@
+module ProjectsList
+  class << self
+    def fetch_section_projects(section)
```

Comment:
style: we prefer `def self.fetch_section_projects(section)` over `class << self` See [style guide](https://github.com/bbatsov/ruby-style-guide#def-self-class-methods).

Along these lines, 
`class << self` shows up [twice](https://github.com/code-dot-org/code-dot-org/search?utf8=%E2%9C%93&q=%22class+%3C%3C+self%22&type=Code) in our codebase vs. [193](https://github.com/code-dot-org/code-dot-org/search?utf8=%E2%9C%93&q=%22def+self.%22&type=Code) for `def self.`